app-emulation/lxd/files/lxd.initd

```diff
@@ -0,0 +1,46 @@
+#!/sbin/openrc-run
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+DAEMON=/usr/sbin/lxd
+PIDFILE=/run/lxd.pid
```

Comment:
ditto, define it in as the `pidfile` parameter. have a loot at `openrc-run (8)`.

Reply:
DAEMON and PIDFILE are fine in this case. We wouldn't be able to get rid of any of the functions even if we did.